src/workerd/io/worker.c++

```diff
@@ -1546,6 +1546,9 @@ Worker::Worker(kj::Own<const Script> scriptParam,
                             } else if (handle == entrypointClasses.workerEntrypoint) {
                               impl->statelessClasses.insert(kj::mv(handler.name), kj::mv(cls));
                               return;
+                            } else if (handle == entrypointClasses.workflow) {
```

Comment:
for the time being, should we gate access to this behind the `experimental` flag?

Reply:
Yes.

Reply:
Very good call out